integrations/clickup/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,46 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: team
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .key
+          title: .name
+          blueprint: '"clickupTeam"'
+          properties:
+            url: (.self | split("/") | .[:3] | join("/")) + "/team/"
+            members: .fields.members
```

Comment:
You're simply grabbing the entire payload for members rather than retrieving the emails. Retrieve the email addresses only

Reply:
done